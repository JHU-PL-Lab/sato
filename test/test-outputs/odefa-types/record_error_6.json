[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "t" ],
          "err_match_val": "true",
          "err_match_expected": "int",
          "err_match_actual": "bool"
        }
      ]
    ],
    "err_input_seq": [ 0 ],
    "err_location": "ret = x + t",
    "err_steps": 245
  }
]