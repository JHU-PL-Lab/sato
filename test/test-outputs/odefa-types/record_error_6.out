Type errors for:
- Input sequence  : [0]
- Found at clause : ret = x + t
--------------------
* Value    : t = true
* Expected : int
* Actual   : bool
1 errors found. No further control flows exist.
